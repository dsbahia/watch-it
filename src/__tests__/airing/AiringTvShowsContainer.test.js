import React from "react";
import { render, screen } from "@testing-library/react";
import AiringTvShowContainer from "../../components/Upcoming/AiringTvShowsContainer";

describe("AiringTvShowsContainer", () => {
  it("should render without errors", () => {
    render(<AiringTvShowContainer />);
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(<AiringTvShowContainer />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("displays a list of top rated movies", async () => {
    render(<AiringTvShowContainer />);

    const movieList = await screen.findByTestId("tv-list");

    expect(movieList).toBeInTheDocument();
  });
});
