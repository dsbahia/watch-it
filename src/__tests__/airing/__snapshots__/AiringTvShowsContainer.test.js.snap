// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AiringTvShowsContainer matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="Watch It Logo"
      class="watch-it-logo"
      src="watch-it.png"
    />
    <div
      class="top-rated-container"
      data-testid="tv-list"
    >
      <div
        class="top-rated-movie-title"
      >
        Airing Tv Shows
      </div>
      <div
        class="top-rated-results"
      />
       
    </div>
  </div>
</DocumentFragment>
`;
